ui/src/app/pages/login/account-creation/account-creation-tos.tsx

```diff
@@ -93,44 +93,21 @@ export const AccountCreationTos = withWindowSize()(
       observer.observe(this.lastPage);
     }
 
+    private setLastPageRef(ref) {
+      this.lastPage = ref;
+    }
+
     render() {
-      const {numPages, loadingPdf, hasReadEntireTos, hasAckedTermsOfService, hasAckedPrivacyStatement} = this.state;
+      const {hasReadEntireTos, hasAckedTermsOfService, hasAckedPrivacyStatement} = this.state;
 
       return <FlexColumn data-test-id='account-creation-tos'
                          style={{flex: 1, padding: '1rem 3rem 0 3rem'}}>
-        {/* TODO: all of this PDF rendering stuff should be broken out into a separate component. */}
-        <div style={{flex: '1 1 0', overflowY: 'auto'}}>
-          {loadingPdf && <SpinnerOverlay/>}
-          <Document data-test-id='tos-pdf-document' file={this.props.pdfPath}
-                    loading=''
-                    onLoadSuccess={data => this.setState(
-                      {numPages: data.numPages, loadingPdf: false})}
-          >
-            {
-              Array.from(
-                new Array(numPages),
-                (el, index) => (
-                  // We can't set inline styles on the react-pdf Page element, so instead we set a
-                  // className and specify some style overrides in src/styles.css
-                  <Page
-                    renderAnnotationLayer={false}
-                    renderTextLayer={false}
-                    loading=''
-                    className='tos-pdf-page'
-                    width={Math.max(500, this.props.windowSize.width * .75)}
-                    key={`page_${index + 1}`}
-                    pageNumber={index + 1}
-                    inputRef={index === numPages - 1 ? (ref) => {
-                      this.lastPage = ref;
-                    } : undefined}
-                    onRenderSuccess={index === numPages - 1 ? () => this.handleLastPageRender() :
-                      undefined}
-                  />
-                ),
-              )
-            }
-          </Document>
-        </div>
+        <PdfViewer
+            handleLastPageRender={() => this.handleLastPageRender()}
+            pdfPath={this.props.pdfPath}
+            setLastPageRef={(ref) => this.setLastPageRef(ref)}
+            windowSize={this.props.windowSize}
```

Comment:
Is it necessary to pass in the windowSize if we're using `withWindowSize()` in the component? Or we can keep this and remove the `withWindowSize()`